services/galley/schema/src/V76_ProposalOrigin.hs

```diff
@@ -0,0 +1,34 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2022 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V76_ProposalOrigin
+  ( migration,
+  )
+where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration =
+  Migration 76 "Add the origin column to the pending proposals table" $
+    schema'
+      [r| ALTER TABLE mls_proposal_refs ADD (
+            origin int
+          )
+        |]
```

Comment:
Resetting the database is not an option. But yes it might break groups that have pending external proposals. Other groups should continue to function: We assume the proposal is not `ProposalOriginClient` if the `origin` is `null`.

Reply:
`Nothing` values will be logged as warning.